vizro-core/tests/unit/vizro/conftest.py

```diff
@@ -13,6 +13,16 @@ def gapminder():
     return px.data.gapminder()
 
 
+@pytest.fixture
+def gapminder_with_datetime():
+    return px.data.gapminder(datetimes=True)
```

Comment:
Similar question as in the features example - do we need both `gapminder` and `gapminder_with_datetime`?